draft-irtf-qirg-principles-09-rc.xml

```diff
@@ -217,13 +217,14 @@
         keywords will be used for the search engine. -->
 
    <abstract>
-     <t>The vision of a quantum internet is to fundamentally enhance Internet
+     <t>The vision of a quantum Internet is to enhance existing Internet
      technology by enabling quantum communication between any two points on
      Earth. To achieve this goal, a quantum network stack should be built from
      the ground up to account for the fundamentally new properties of quantum
-     entanglement. The first realisations of quantum entanglement networks are
-     imminent, but there is no practical proposal for how to organise, utilise,
-     and manage such networks. In this memo, we attempt to lay down the
+     entanglement. The first quantum entanglement networks have been realised
+     <xref target="Pompili21" />, but there is no practical proposal for how to
```

Comment:
Interesting paper; thanks for pointing to it. I can see how the approach described in this paper allows for a rich topology without using memories or swapping, but I do not see how it overcomes the distance limitations of existing QKD (~ 120 km for BB84, ~ 600 km for TF).  The "long distances" referred to in this paper (start of page 4) are only up to 12.6 km.  Anyway... since you said "no need to act on this, leaving it to the mailing list" I am marking the comment as resolved.